src/Components/Answer/Answer.tsx

```diff
@@ -1,13 +1,15 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { Link } from "react-router-dom";
 
 export type AnswerProps = {
   votes: number;
-  question: string;
+  question?: string | null | undefined;
```

Comment:
why is question optional

Reply:
the answer component has two usages , one is with question and one is without